### PR TITLE
[production/RRFS.v1] Update task names for forecast, post, and prdgen tasks

### PR DESCRIPTION
**Pull request by BenjaminBlake-NOAA**
## DESCRIPTION OF CHANGES: 

- In this PR, the task names for 3 jobs are updated: run_fcst -> forecast, run_post -> post, run_prdgen -> prdgen.  The J-jobs, ex-scripts, and task-related variable names have been updated accordingly throughout the workflow.

## TESTS CONDUCTED: 

I ran a cycle of the fire weather nest to test the forecast, post, and prdgen tasks.  I also successfully built the workflow using FV3LAM_wflow_nonDA.xml and FV3LAM_wflow.xml to make sure I didn't make any mistakes/typos with these changes.

### Machines/Platforms:

- WCOSS2
  - [x] Cactus/Dogwood
  - [ ] Acorn
- RDHPCS
  - [ ] Hera
  - [ ] Jet
  - [ ] Orion
  - [ ] Hercules

### Test cases: 

- [ ] Engineering tests
  - [ ] Non-DA engineering test
  - [ ] DA engineering test
    - [ ] Retro
    - [ ] Ensemble
    - [ ] Parallel
- [x] RRFS fire weather
- [ ] RRFS_A:
- [ ] RRFS_B:
- [ ] RTMA:
- [ ] Others:

## ISSUE: 

- This PR is the start of the work to address the task name changes outlined in issue #247

### CODE REVIEW

File: ush/config_defaults.sh
Comment (MatthewPyle-NOAA) [inline ush/config_defaults.sh:1]: Nothing to do with your changes, but at line 1315 there is a "the the".

File: ush/sample_configs/non-DA_eng/config.nonDA.grib2.wcoss2.sh
Comment (MatthewPyle-NOAA) [inline ush/sample_configs/non-DA_eng/config.nonDA.grib2.wcoss2.sh:1]: RUN_TASK_PRDGEN seems to be new here (not replacing a RUN_TASK_RUN_PRDGEN entry).

**Review by MatthewPyle-NOAA — COMMENTED** [review]
One trivial comment, and one meaningful one.  Should easily to mergeable tomorrow.

File: ush/config_defaults.sh
Comment (BenjaminBlake-NOAA) [reply ush/config_defaults.sh:1]: Thanks, I just fixed that.

File: ush/sample_configs/non-DA_eng/config.nonDA.grib2.wcoss2.sh
Comment (BenjaminBlake-NOAA) [reply ush/sample_configs/non-DA_eng/config.nonDA.grib2.wcoss2.sh:1]: Yes it is!  I noticed RUN_TASK_PRDGEN was included in all the other config files in the non-DA_eng directory except for this one (grib2, wcoss2) so I added it.  It is set to false in all of these files, which is also the default value, so it shouldn't affect anything.

Comment (MatthewPyle-NOAA) [reply ush/sample_configs/non-DA_eng/config.nonDA.grib2.wcoss2.sh:1]: Got it - thanks!

**Review by MatthewPyle-NOAA — APPROVED** [review]
For grins I ran a non-DA engineering test on Hera, and it ran okay.  But I'll be submitting a PR shortly to update the model version for this branch  (I needed to do that to compile on Hera).

**Pull request merged**